src/cmds/module/logmod.c

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file logmod.c
+ * @brief
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version
+ * @date 24.03.2020
+ */
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <framework/mod/api.h>
+#include <framework/mod/integrity.h>
+#include <framework/mod/types.h>
+#include <util/log.h>
+
+static void print_usage(void) {
+	printf("Usage: logmod [name [level]]\n");
+}
+
+int main(int argc, char **argv) {
+	const struct mod *mod;
+	struct logger *logger;
+	int mod_found = 0;
+	int opt;
+
+	if (argc < 2) {
+		print_usage();
+		return 0;
+	}
+
+	while (-1 != (opt = getopt(argc, argv, "h"))) {
+		switch (opt) {
+		case 'h':
+			print_usage();
+			return 0;
+		case '?':
+			break;
+		default:
+			return -EINVAL;
+		}
+	}
+
+	mod_foreach(mod) {
+		if (!strstr(mod_name(mod), argv[1])) {
+			continue;
+		}
```

Comment:
Why do not strcmp?

Also, Embox suffers from not very good namings like `core`, `all`, `arch`. At the same time, we do not forbid to have several modules with the same names in different packages, so formally it is ok. Maybe `package_name.mode_name` would be more deterministic?